app/src/main/kotlin/com/saltedge/authenticator/widget/security/LockableActivity.kt

```diff
@@ -168,6 +166,25 @@ abstract class LockableActivity :
 
     override fun onNewPasscodeConfirmed(passcode: String) {}
 
+    override fun onResetPasscode() {
+        getUnlockAppInputView()?.let {
+            it.setInputViewVisibility(show = false)
+            it.setResetPasscodeView()
+            it.showErrorMessage(false)
+        }
+    }
+
+    override fun showDeleteConnectionView(requestCode: Int) {
```

Comment:
`requestCode` is not used

Reply:
as i see you don't have delete connection view. you ask to delete all data. fix naming